### PR TITLE
Feature/#63 states of measurables

### PR DESCRIPTION
**Pull request by GalochkinKirov**

### CODE REVIEW

File: server/src/async/events.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const EVENTS = {
+  MEASURABLES_STATES: 'MEASURABLES_STATES',
```
Comment (OAGr) [inline server/src/async/events.js]: If this is an EVENT, can we change it to something like, ``RUN_MEASURABLE_STATE_TRANSITIONS``

File: server/src/async/listeners.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+const emitter = require('./emitter');
+const events = require('./events');
+
+const triggers = require('./triggers');
+
+function measurablesStates() {
+  try {
+    console.log('Job::measurablesStates');
```
Comment (OAGr) [inline server/src/async/listeners.js]: please rename this as well.

File: server/src/async/listeners.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
+const emitter = require('./emitter');
+const events = require('./events');
+
+const triggers = require('./triggers');
+
+function measurablesStates() {
```
Comment (OAGr) [inline server/src/async/listeners.js]: rename to be more like an action

File: server/src/async/index.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
+const cron = require('node-schedule');
+
+const emitter = require('./emitter');
+const events = require('./events');
+
+function runJobs() {
+  console.log('Jobs are in a queue.');
+
+  const EVERY_DAY_AT_MIDNIGHT = '0 0 * * *';
```
Comment (OAGr) [inline server/src/async/index.js]: Can we run this every hour instead? Also, is it possible to trigger remotely for testing?

File: server/src/models-abstract/measurable-model.js [hunk 5]
```diff
@@ -25,6 +29,26 @@ class MeasurableModel extends ModelPostgres {
     );
   }
 
+  /**
+   * @return {Promise<boolean>}
+   */
+  setJudgementPending() {
+    return this.model.update({
```
Comment (OAGr) [inline server/src/models-abstract/measurable-model.js]: I kind of hate to say this, but can we move this logic up out of the db layer and into the model layer or similar? So the model layer has a method for ``shouldupdateStateToJudgementPending``, and another for ``updateStateToJudgementPending``. That way we could later have hooks here to add notifications, and other things. For instance, the ``resulution.js`` bot should run here.

File: server/src/models/measurable.js [hunk 6]
```diff
@@ -35,24 +37,12 @@ module.exports = (sequelize, DataTypes) => {
...
     },
-    resolutionEndpoint: {
-      type: DataTypes.TEXT,
-      allowNull: true,
-    },
-    hasResolutionEndpointResolved: {
-      type: DataTypes.BOOLEAN,
-      allowNull: false,
-      defaultValue: false,
-    },
```
Comment (OAGr) [inline server/src/models/measurable.js]: Why did you move these down?

File: server/src/async/events.js (hunk 1 above)
Comment (GalochkinKirov) [reply server/src/async/events.js]: Event it is not a command.

Comment (OAGr) [reply server/src/async/events.js]: Even so. Even ``MEASURABLE_STATE_TRANSITIONS`` would be more specific.

Comment (GalochkinKirov) [reply server/src/async/events.js]: Thanks, ok.
